### ISSUE TITLE
Is there a way to do this using the API?

### ISSUE THREAD

**Issue by JeffreyShran** [defparam/h1passets#3]
I'm wondering if this could be done using the API instead so we could use a permanent authorisation token?

I did see something about [structured scope](https://api.hackerone.com/reference/#structured-scope) but I'm not sure if its possible.

**Comment by defparam** [comment]
I'm actually not sure either, I assumed the formal API was intended for paid program maintainers instead of researchers (but I could be wrong). I'll be completely honest, I wrote this in like 1-2 hours because I needed to scrape all my private programs and I figured this would be useful to other people. I fully expect the GraphQL commands to change and eventually break this. At which point I'll probably just patch it. For more formal compatibility I urge others to fork and provide pull requests or create a new script altogether, I have limited bandwidth for any other changes past normal functionality

extra points to anyone who wants to fork this: I would love to see bugcrowd support too =)

**Comment by JeffreyShran** [comment]
Yeah I totally get that and appreciate you making the script available. I'm 99% sure I saw something similar somewhere now I think more about it. If there's a way I'd certainly look at contributing to some work.
